reference/forms/types/options/choice_translation_domain.rst.inc

```diff
@@ -0,0 +1,15 @@
+choice_translation_domain
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. versionadded:: 2.7
```

Comment:
Forget my previous comment. In 2.8 we added support for this option in date-related types.